### ISSUE TITLE
Unable to sync github repos for new account

### ISSUE THREAD

**Issue by benasher44** [travis-ci/travis-ci#5795]
Hi! I've just setup my github account with travis-ci, and I've successfully given travis-ci access to my github account. However, when I click the "Sync Account" button on my profile page, it says "Syncing from Github" and then finishes with this error:

"Sorry, it seems like we couldn't find any repositories you have access to on GitHub."

Is there anything I'm missing here? Thanks!

Ben

**Comment by flipjs** [comment]
I have the same issues as well. It is stuck on the 'One more thing' screen. If I go to my profile, there's a loading gif that says Syncing from Github and at the botto, it says 'Sorry, it seems like we couldn't find any repositories you have access to on GitHub.'

On Github integrations, it says Travis CI was added. I also added Travis on one of my repo. I created an organization too, with third-party access set to no restrictions, and member priveleges set to admin. My Travis CI profile still says 'You are not currently a member of any organization.', but I already created one.

How long does it take to synchronize repositories? I only have like 50 or so, and most of them are really small projects.

Tried the signing out and signing back in, as others suggested, but I still get the same notice that it couldnt find any repositories.
